templates/emails/report_new.html

```diff
@@ -0,0 +1,27 @@
+{% extends "emails/base.html" %}
+{% load mail_tags %}
+{% autoescape off %}
+{% block title %}{{settings.MAIN_DOMAIN}} - New Moderation Report{% endblock title %}
+{% block subtitle %}{{settings.MAIN_DOMAIN}} - New Moderation Report{% endblock subtitle %}
+
+{% block body_greeting %}
+    {% email_body_content "Moderation Team," %}
+    {% email_body_content "A new report has been recieved for {{config.site_name}}." %}
+{% endblock body_greeting %}
+
+{% block body_content %}
+    {% email_body_content "=============== Report Summary ===============" %}
+    {% email_body_content "Reporter Identity: {{report.source_identity}} @ {{report.source_domain}}" %}
```

Comment:
`report.source_identity` can be None if it's an anonymous report from another domain, so we should handle that.

Reply:
oh. I've never seen an option for anonymous reporting 🤔 Sound interesting

Reply:
Updated to "Anonymous Report from {{report.source_domain}}" if no source_identity if that works?